plutus-core/test/Check/Equality.hs

```diff
@@ -0,0 +1,182 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Check.Equality
+    ( test_equality
+    ) where
```

Comment:
Moved equality tests from the outermost `Spec`, 'cause there are enough of them to keep them separately.